lib/CMakeLists.txt

```diff
@@ -267,6 +267,11 @@ if(neon_compile_result)
     if (have_neonv8_result)
         OVERRULE_ARCH(neonv7 "CPU is armv8")
     endif()
+
```

Comment:
According to @balister ARM CPUs may support both architectures. In this case, I'd suggest to separately check for both architectures and call `OVERRULE_ARCH` on each one separately. We should be able to have both architectures compiled and let VOLK choose the preferred one according to `volk_profile` results.

Reply:
I think you misunderstood what I wrote, or I wrote the wrong thing. I can not imagine a situation where compile test for neonv7 and neonv8 both succeed.

Reply:
This solution looks good to me now. @balister, do you agree too?

Reply:
This solution LGTM now